packages/web-forms/src/components/FormPanel.vue

```diff
@@ -69,23 +69,27 @@ const toggleMenu = (event:  Event) => {
 </template>
 
 <style scoped lang="scss">
+.chevron {
+	padding-top: 2px;
+}
+
```

Comment:
Quite a lot of what's going on in here feels like it might be less fussy and less full of magic tiny pixel spacing if the containers of affected elements were flexbox/vertically centered. I don't know if it'd cover everything changing here, but it did seem to have a similar effect in a couple places where I tried it.